homeassistant/helpers/entity.py

```diff
@@ -64,6 +64,47 @@
 FLOAT_PRECISION = abs(int(math.floor(math.log10(abs(sys.float_info.epsilon))))) - 1
 
 
+class ContextFilter(Protocol):
```

Comment:
Do we need to overengineer it like this. Shouldn't we just start with a single expected state or a tuple of expected states? We can overload it later.